### DISCUSSION TITLE
I have some questions about applying for github education.

### DISCUSSION THREAD

**Discussion by tengo-cpu** [community/community#144269] [GitHub Education]
### Hi everyone,

1. I can't fill in the first character of the Chinese name.
It shows that the name is too short
![1 (2)](https://github.com/user-attachments/assets/cc442812-b40f-404c-8c97-821c8c486fb0)

2. I have a student ID card, but there is no date on it, can I provide proof of payment for this semester, the date of payment is in Taiwan's Republic of China (R.O.C.) calendar year?
I took a photo of my payment certificate and my student ID in the same photo.

3. "You are unlikely to be verified until you have completed your GitHub user profile with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying."
![2](https://github.com/user-attachments/assets/120995d7-5fad-45ff-ac16-8ce7bc0ba4a4)

I don't understand where I put my name wrong.
I can't find the reason.

**Comment by huddmax** [comment]
Hi there!

Here are some suggestions that might help with each of your issues:

1. **Chinese Name Input**: It sounds like GitHub's system may have trouble recognizing the character length in certain languages. You could try reaching out to GitHub Support directly to explain that this is your real name in Chinese, and the character length requirement is blocking it. They might be able to override this requirement or suggest an alternative solution.

2. **Student ID without a Date**: If your student ID doesn’t show a date, providing proof of payment for the current semester is a good idea. The GitHub Education team generally accepts official documents that show you’re currently enrolled, so a payment receipt should work as long as the date is clear, even if it's in the R.O.C. calendar year. Just make sure the document clearly shows your name and the current academic term.

3. **Full Name Verification**: For the name issue, double-check that the name on your GitHub profile exactly matches the one on your academic documents, including any middle names or special characters. Sometimes, even small differences or missing parts of your name can cause verification issues. After updating your profile, log out of GitHub and log back in before reapplying to make sure the changes take effect.

Hopefully, these steps help you get through the verification process. Good luck!

**Reply by tengo-cpu** [reply]
I have entered the name, and I also photographed the proof of enrollment and student ID, are clearly visible, I do not understand what you are engaged in, I am in line with the requirements, try to do a good job, but also has been rejected, you have been a mechanical fuking reply!

fuk github education....

If you don't want people to use it, don't open it, I even went to the school today to apply for a paper certificate of attendance, nah fuk you, tried for hours and hours, spent time and all I got was canned replies!

**Comment by tengo-cpu** [comment]
I have entered the name, and I also photographed the proof of enrollment and student ID, are clearly visible, I do not understand what you are engaged in, I am in line with the requirements, try to do a good job, but also has been rejected, you have been a mechanical fuking reply!

fuk github education....

If you don't want people to use it, don't open it, I even went to the school today to apply for a paper certificate of attendance, nah fuk you, tried for hours and hours, spent time and all I got was canned replies!

**Reply by tengo-cpu** [reply]
Ruined my mood for the rest of the day. Fuck you.

**Reply by tengo-cpu** [reply]
It's been a day. I can't get through to customer service. Fuk you.Always a canned reply

**Reply by tengo-cpu** [reply]
I offer
1.School Mailbox
2.Real Letter Name
3.Student ID
4.Proof of payment for the semester
5.Proof of enrollment
Fuk you github education, still won't let me pass.
Fuk you and your mom for not giving me a pass and not having any official recommendation.
I've been working on this for hours. I'm really pissed off.

**Comment by tengo-cpu** [comment]
<html>
<body>

Hey 翁敬宸,Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Nov 13, 2024 for @tengo-cpu.Please ensure that your academic affiliation document contains your last name exactly as it appears in your GitHub billing information. If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.Please ensure that your academic affiliation document contains your first name exactly as it appears in your GitHub billing information. Variations of your name and nicknames are unacceptable.You are unlikely to be verified until you have completed your GitHub user profile with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.Once you have resolved the issues with your sumbission, visit us again and reapply.Have questions? Please seek help via our Student Developer Pack Application FAQ or visit the GitHub Education Community.💖 The GitHub Education Team
--



</body>
</html>Hey 翁敬宸,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Nov 13, 2024 for [@tengo-cpu](https://github.com/tengo-cpu).

Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). Variations of your name and nicknames are unacceptable.

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.



Once you have resolved the issues with your sumbission, [visit us again and reapply](https://education.github.com/).

Have questions? Please seek help via our [Student Developer Pack Application FAQ ](https://github.com/orgs/community/discussions/111352)or visit the [GitHub Education Community.](https://github.com/orgs/community/discussions/categories/github-education)

💖 The GitHub Education Team

**Comment by tengo-cpu** [comment]
I offer
1.School Mailbox 
2.Real Letter Name
3.Student ID
4.Proof of payment for the semester
5.Proof of enrollment
Fuk you github education, still won't let me pass.
Fuk you and your mom for not giving me a pass and not having any official recommendation.
I've been working on this for hours. I'm really pissed off.

**Comment by 1817268089** [comment]
I've been messing with it all day too, and it's the same thing after every submission, so fuck me too
